client/analytics/report/style.scss

```diff
@@ -42,4 +42,14 @@
 			font-weight: 600;
 		}
 	}
+
+	.woocommerce-table__header .components-button.is-default {
+		color: $gray-text;
+		border-color: #ccc;
+		background: #f7f7f7;
+	}
+
+	.woocommerce-table__header .components-button.is-default:hover {
+		color: $gray-text;
+	}
```

Comment:
This button has a disabled state that is no longer:

## Master

<img width="210" alt="Screen Shot 2019-10-08 at 3 18 17 PM" src="https://user-images.githubusercontent.com/1922453/66362565-c85c9400-e9df-11e9-8313-1f72b94798f3.png">

## This branch

<img width="194" alt="Screen Shot 2019-10-08 at 3 10 40 PM" src="https://user-images.githubusercontent.com/1922453/66362576-d14d6580-e9df-11e9-8499-fbb5b37c4eb5.png">

Reply:
Thanks, I've fixed all the buttons with some better rules in the global stylesheet, so this is fixed now.